revokefs/Makefile.am.inc

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2016 Colin Walters <walters@verbum.org>
+# Copyright (C) 2018 Alexander Larsson <alexl@redhat.com>
+#
+# SPDX-License-Identifier: LGPL-2.0+
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
+
+bin_PROGRAMS += revokefs-fuse
```

Comment:
Hmm, i think we should maybe put this in libexec_PROGRAMS instead to avoid polluting PATH.

Reply:
I am taking this `maybe` as a yes. Changed `bin_PROGRAMS` to `libexec_PROGRAMS` in the recent round of iteration.